packages/grid/src/vaadin-grid-drag-and-drop-mixin.js

```diff
@@ -371,11 +374,11 @@ export const DragAndDropMixin = (superClass) =>
      * the conditions change.
      */
     filterDragAndDrop() {
-      Array.from(this.$.items.children)
-        .filter((row) => !row.hidden)
-        .forEach((row) => {
```

Comment:
Changed the code here and elsewhere to use `forEach` and `if` instead of `filter()` + `forEach()`.